README.md

```diff
@@ -97,7 +97,302 @@ Where:
 - `--package-only`: create the plugins wheels, sha256 file, helm-chart tarball
   and package the entire application into a tarball.
 
-## Detailed instructions
+Below you will find detailed instructions on how to use this repo to create
+a StarlingX App package from your own code.
 
-TODO: add the AppGenGuide from https://github.com/bmuniz-daitan/poc-starlingx-messages
-when ready.
\ No newline at end of file
+# Deploy an application as a StarlingX app
```

Comment:
Don't you think we can integrate better this section into the README? What I mean is, it seems kinda lost in here in some places saying the same thing.